### DISCUSSION TITLE
After nextcloudpi instalation

### DISCUSSION THREAD

**Discussion by Serhioromano** [tteck/Proxmox#2255] [Ideas]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

After installation when I go to administrator settings I have those

Security & setup warnings
It's important for the security and performance of your instance that everything is configured correctly. To help you with that we are doing some automatic checks. Please see the linked documentation for more information.

There are some errors regarding your setup.
The PHP memory limit is below the recommended value of 512MB.
Your installation has no default phone region set. This is required to validate phone numbers in the profile settings without a country code. To allow numbers without a country code, please add "default_phone_region" with the respective [ISO 3166-1 code ↗](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements) of the region to your config file.
The PHP OPcache module is not properly configured. See the [documentation ↗](https://docs.nextcloud.com/server/27/go.php?to=admin-php-opcache) for more information.
The OPcache interned strings buffer is nearly full. To assure that repeating strings can be effectively cached, it is recommended to apply opcache.interned_strings_buffer to your PHP configuration with a value higher than 12.
The PHP module "imagick" is not enabled although the theming app is. For favicon generation to work correctly, you need to install and enable this module.
Please double check the [installation guides ↗](https://docs.nextcloud.com/server/27/go.php?to=admin-install), and check for any errors or warnings in the [log](https://nextcloudpi.local/index.php/settings/admin/logging).

Is it possible to make PHP configured already with all those notes and no need for additional configuration?

[2023-12-27_15-41-08](https://github.com/tteck/Proxmox/assets/650741/d3a5c890-475f-4734-9f4d-12ddb0e7a97c)

### Which Linux distribution are you employing?

Debian 11

**Comment by tteck** [comment]
> - For suggestions, questions or feature/script requests, please share them in the [Discussions section.](https://github.com/tteck/Proxmox/discussions)

**Comment by tteck** [comment]
Can you explain what steps you're taking to comply with the warnings?

**Reply by Serhioromano** [reply]
I just runned suggested command to create LXC 

```bash
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nextcloudpi.sh)"
```

Then immediately login and go to that information section.
